pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go

```diff
@@ -0,0 +1,271 @@
+package openshiftkubeapiserver
+
+import (
+	"fmt"
+	"net"
+	"sort"
+
+	"io/ioutil"
+
+	"bytes"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
+	configapilatest "github.com/openshift/origin/pkg/cmd/server/apis/config/latest"
+	originadmission "github.com/openshift/origin/pkg/cmd/server/origin/admission"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func ConfigToFlags(kubeAPIServerConfig *configapi.MasterConfig) ([]string, error) {
+	args := map[string][]string{}
+	for key, slice := range kubeAPIServerConfig.KubernetesMasterConfig.APIServerArguments {
+		for _, val := range slice {
+			args[key] = append(args[key], val)
+		}
+	}
+
+	host, portString, err := net.SplitHostPort(kubeAPIServerConfig.ServingInfo.BindAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	// these flags are overridden by a patch
```

Comment:
who maintains this list? not sure of the value

Reply:
> who maintains this list? not sure of the value

We need this list so that we can create a struct to represent it in order to manage it.  We'll need to write a test of some sort to make sure we don't lose them.